Bottom: 32611dc98d9d7607404e6f4822342e3700eb389c
Top:    b8a0769bc8d4bdc6c71e33aeee26cb00c5af43c5
Author: Ben Greear <greearb@candelatech.com>
Date:   2014-02-12 17:52:56 -0800

ath10k: do not ratelimit ath10k_warn messages

These are too important to miss, and if we ever do have some
that are overly verbose, could ratelimit the call to ath10k_warn
instead of doing the ratelimit inside of ath10k_warn.

Signed-off-by: Ben Greear <greearb@candelatech.com>


---

diff --git a/drivers/net/wireless/ath/ath10k/debug.c b/drivers/net/wireless/ath/ath10k/debug.c
index a282a5a..d4b52d0 100644
--- a/drivers/net/wireless/ath/ath10k/debug.c
+++ b/drivers/net/wireless/ath/ath10k/debug.c
@@ -239,7 +239,7 @@ void ath10k_warn(struct ath10k *ar, const char *fmt, ...)
 
 	va_start(args, fmt);
 	vaf.va = &args;
-	dev_warn_ratelimited(ar->dev, "%pV", &vaf);
+	dev_warn(ar->dev, "%pV", &vaf);
 	trace_ath10k_log_warn(ar, &vaf);
 
 	va_end(args);
